LDM-148.tex

```diff
@@ -332,21 +333,26 @@ \section{Component Overview}\label{component-overview}
 \label{fig:scipi}
 \end{figure}
 
-The middleware layer includes certain supporting software products and the Batch Production software products:
+The middleware layer includes software products that allow the building of algorithmic pipelines and abstract their data access.
 \begin{itemize}
 \item
   Data access client (Data Butler) (\texttt{daf\_persistence})
 \item
-  Parallel distributed database (\texttt{qserv})
+  Task framework (\texttt{pex\_*}, \texttt{log}, \texttt{pipe\_base}, \texttt{ctrl\_pool})
+\end{itemize}
+
+These pipelines are in turn executed and controlled by Batch Production software products:
+\begin{itemize}
 \item
-  Task framework (\texttt{pex\_*}, \texttt{log}, \texttt{pipe\_base},
-		\texttt{pipe\_supertask}, \texttt{ctrl\_pool})
+  Campaign management (\texttt{ctrl\_bps})
 \item
-  Campaign management and workflow for production control (\texttt{ctrl\_*})
+  Workload/workflow
```

Comment:
Feels like this is missing a word — “Workload/workflow” isn't a product, but “Workload/workflow management” (or something like that) is.